friends/src/AddFriend.js

```diff
@@ -0,0 +1,91 @@
+// Import
+
+import React, { useState } from "react";
+
+import  { axiosWithAuth }from "./axiosWithAuth";
+
+function AddFriend(props){
+    const [name, setName] = useState({
+        name: "",
+        age: "",
+        email: "",
+        id: Date.now()
+    });
+
+    const changeHandler = e => {
+        //e.preventDefault();
+        setName({ ...name, [e.target.name]: e.target.value });
+    };
+ 
+    function addFriend(e) {
+      //e.preventDefault();
+      axiosWithAuth()
+        .post("http://localhost:5000/api/friends", name)
+        .then(res =>{
+
+        console.log('friends', res.data)
+
```

Comment:
I see you are posting to the server but you are not updated state... this can be accomplished by passing down your friends state from your friendslist component or using your state hooks :)